mobile/src/main/java/com/wedriveu/mobile/service/scheduling/SchedulingService.java

```diff
@@ -0,0 +1,26 @@
+package com.wedriveu.mobile.service.scheduling;
+
+import com.google.android.gms.location.places.Place;
+import com.wedriveu.mobile.util.location.LocationServiceListener;
+
+/**
+ * This service interacts with the VehicleService Microservice and asks it to find a proper vehicle based on the coordinates
```

Comment:
I generally use {@linkplain className} but in this context I still don't know how the vehicle service will be called in the code, as it has no reference to it yet.